examples/cd_context_20newsgroup.ipynb

```diff
@@ -0,0 +1,3186 @@
+{
```

Comment:
Apologies, a slightly awkward new convention is that the actual example notebooks should be saved in `doc/source/examples/`. Running `make build_docs` will then create symbolic links to these in `examples/`. This is because the way we used to do it before (using `nbsphinx_link`) prevented us doing incremental docs builds.